install.sh

```diff
@@ -994,11 +994,27 @@ case "${SHELL}" in
     ;;
 esac
 
-cat <<EOS
+in_shell_profile=$(grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}")
```

Comment:
`grep -qs` suppresses _all_ output, so `in_shell_profile` is always set to the empty string, which explains the "command not found" error you're seeing.

```suggestion
in_shell_profile=$(grep -s "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}")
```